src/actions/player/playerRecordsActions.js

```diff
@@ -40,14 +40,16 @@ export const getPlayerRecords = (playerId, options = {}) => (dispatch, getState)
 
   return fetch(`${API_HOST}${getUrl(playerId, options, url)}`, { credentials: 'include' })
     .then(response => response.json())
-    .then(json => Object.keys(json).map(key => ({
+    .then(json => Object.keys(json)
+    .map(key => ({
       name: key,
       value: json[key][key],
       hero_id: json[key].hero_id,
       start_time: json[key].start_time,
       match_id: json[key].match_id,
       game_mode: json[key].game_mode,
-    })))
+    }))
+    .filter(key => key.value !== null))
```

Comment:
Technically this isn't a key anymore since we mapped it into an array of objects.  Maybe "record" or "element" is a better name.

Also check for undefined.